crates/l2/sequencer/configs.rs

```diff
@@ -0,0 +1,49 @@
+use ethrex_common::{Address, U256};
+use secp256k1::SecretKey;
+use std::net::IpAddr;
+
+pub struct SequencerConfig {
+    // pub proposer: ProposerConfig,
```

Comment:
Should this be removed?

Reply:
Yep. Good catch.

Reply:
Done in [43fd4a8](https://github.com/lambdaclass/ethrex/pull/2606/commits/43fd4a8e4d1b981d778b6024a7e88c93fda632b3).